src/common/playlist/playlist-manager.js

```diff
@@ -34,14 +36,18 @@ class PlaylistManager {
         this._player.dispatchEvent(new FakeEvent(PlaylistEventType.PLAYLIST_LOADED, {playlist: this._playlist}));
         const next = this._playlist.next;
         if (next.item) {
-          this._setItem(next.item, next.index);
+          this._setItem(next.item, next.index).then(() => {
+            this._eventManager.listenOnce(this._player, PlaylistEventType.PLAYLIST_ITEM_CHANGED, () => {
+              this._player.configure({playback: {autoplay: this._options.autoContinue}});
```

Comment:
wouldn't that cause each call to configure to do this? so calling configure multiple times will cause this config to fire.

Reply:
fixed. 10x.
new it's only when new sources configured